[Service Bus] Operation Options Tracing getNamespaceProperties with tracing failing in nightly runs

Service Bus nightly test runs are failing with:

> Error message:
>expected { Object (roots) } to deeply equal { Object (roots) }
> 
> Stack trace:
>AssertionError: expected { Object (roots) } to deeply equal { Object (roots) }
    at Context.<anonymous> (/Users/runner/work/1/s/sdk/servicebus/service-bus/test/internal/operationOptionsForATOM.spec.ts:279:14)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)

For more details check here:

- https://dev.azure.com/azure-sdk/internal/_build/results?buildId=1304697&view=results
- https://dev.azure.com/azure-sdk/internal/_build/results?buildId=1307991&view=results

--- a/sdk/core/core-http/CHANGELOG.md
+++ b/sdk/core/core-http/CHANGELOG.md
@@ -8,6 +8,9 @@
 
 ### Bugs Fixed
 
+- Updated our tracing span names to conform to the [OpenTelemetry Specification](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/http.md#name)
+  - New HTTP spans will use the `HTTP <VERB>` convention instead of using the URL path.
+
 ### Other Changes
 
 ## 2.2.3 (2022-01-06)

--- a/sdk/core/core-http/CHANGELOG.md
+++ b/sdk/core/core-http/CHANGELOG.md
@@ -8,7 +8,7 @@
 
 ### Bugs Fixed
 
-- Updated our tracing span names to conform to the [OpenTelemetry Specification](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/http.md#name)
+- Updated the HTTP tracing span names to conform to the [OpenTelemetry Specification](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/http.md#name). [#19838](https://github.com/Azure/azure-sdk-for-js/pull/19838)
   - New HTTP spans will use the `HTTP <VERB>` convention instead of using the URL path.
 
 ### Other Changes

--- a/sdk/core/core-rest-pipeline/CHANGELOG.md
+++ b/sdk/core/core-rest-pipeline/CHANGELOG.md
@@ -8,6 +8,9 @@
 
 ### Bugs Fixed
 
+- Updated our tracing span names to conform to the [OpenTelemetry Specification](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/http.md#name)
+  - New HTTP spans will use the `HTTP <VERB>` convention instead of using the URL path.
+
 ### Other Changes
 
 ## 1.4.0 (2022-01-06)

--- a/sdk/core/core-rest-pipeline/CHANGELOG.md
+++ b/sdk/core/core-rest-pipeline/CHANGELOG.md
@@ -8,7 +8,7 @@
 
 ### Bugs Fixed
 
-- Updated our tracing span names to conform to the [OpenTelemetry Specification](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/http.md#name)
+- Updated the HTTP tracing span names to conform to the [OpenTelemetry Specification](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/http.md#name). [#19838](https://github.com/Azure/azure-sdk-for-js/pull/19838)
   - New HTTP spans will use the `HTTP <VERB>` convention instead of using the URL path.
 
 ### Other Changes

--- a/sdk/eventgrid/eventgrid/test/public/eventGridClient.spec.ts
+++ b/sdk/eventgrid/eventgrid/test/public/eventGridClient.spec.ts
@@ -239,7 +239,6 @@ describe("EventGridPublisherClient", function (this: Suite) {
       assert.equal(spans.length, 3);
       assert.equal(spans[0].name, "root");
       assert.equal(spans[1].name, "Azure.Data.EventGrid.EventGridPublisherClient-send");
-      assert.equal(spans[2].name, "HTTP POST");
 
       resetTracer();
     });

--- a/sdk/eventgrid/eventgrid/test/public/eventGridClient.spec.ts
+++ b/sdk/eventgrid/eventgrid/test/public/eventGridClient.spec.ts
@@ -239,7 +239,7 @@ describe("EventGridPublisherClient", function (this: Suite) {
       assert.equal(spans.length, 3);
       assert.equal(spans[0].name, "root");
       assert.equal(spans[1].name, "Azure.Data.EventGrid.EventGridPublisherClient-send");
-      assert.equal(spans[2].name, "/api/events");
+      assert.equal(spans[2].name, "HTTP POST");
 
       resetTracer();
     });

--- a/sdk/servicebus/service-bus/test/internal/operationOptionsForATOM.spec.ts
+++ b/sdk/servicebus/service-bus/test/internal/operationOptionsForATOM.spec.ts
@@ -264,7 +264,7 @@ describe("Operation Options", () => {
                     children: [
                       {
                         children: [],
-                        name: "/$namespaceinfo",
+                        name: "HTTP GET",
                       },
                     ],
                     name: "Azure.ServiceBus.ServiceBusAdministrationClient-getResource",

